pkg/oc/bootstrap/docker/up.go

```diff
@@ -1180,21 +1213,6 @@ func (c *ClientStartConfig) InstallMetrics(out io.Writer) error {
 	return c.OpenShiftHelper().InstallMetrics(f, openshift.MetricsHost(c.RoutingSuffix, c.ServerIP), c.Image, c.ImageVersion)
 }
 
-// CheckServiceCatalogPrereqVersion ensures the OpenShift server version is high enough to
-// run the service catalog.
-func (c *ClientStartConfig) CheckServiceCatalogPrereqVersion(out io.Writer) error {
```

Comment:
shouldn't this be separate with SC removal?

Reply:
> shouldn't this be separate with SC removal?

This is just a version check, not the SC installation.  We don't support running 3.7 with 3.9 or 3.10, so we don't need the check anymore.